### ISSUE TITLE
[CoE Starter Kit - BUG] Version issue when opening Sept CoE Dashboard PBI report

### ISSUE THREAD

**Issue by peteredc** [microsoft/coe-starter-kit#3655]
### Describe the issue

When opening CoE Dashboard for Sept in Power BI desktop I get the below error.  I uninstalled and reinstalled from MS site latest version of PBI.  No change.

![image](https://user-images.githubusercontent.com/93291159/189767777-e99c97ce-65a4-40ef-9b9c-7175f0f333f8.png)

### Expected Behavior

Report opens with no warning/error of version mismatch.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.45

### What app or flow are you having the issue with?

Power BI Dashboard

**Comment by Jenefer-Monroe** [comment]
If you just close and proceed, does it work? There isnt much we can do about the product giving that message in error.

**Comment by damienlong** [comment]
I initially didn't have the issue as I had the September 2022 build of Power BI Desktop. I uninstalled and installed latest and ended up with the August 2022 build of Power BI Desktop. With this version of Power BI Desktop, I was able to reproduce the customer's issue. The report does not load queries. Question is, how do we upgrade from August version to September version of Power BI Desktop?

![image](https://user-images.githubusercontent.com/30066912/189833214-d7a08467-a368-4e3c-a875-1049c4eb4417.png)

**Comment by peteredc** [comment]
Interesting! I checked the version installed, and it is definitely the August version.

To add to your question, where is the Sept PBI? Where did you get your Sept version from?

I downloaded from here using the advanced download, not the link that takes you to MS Store. https://powerbi.microsoft.com/en-us/downloads/

![image](https://user-images.githubusercontent.com/93291159/189901511-49e4cb49-46ed-4998-98f8-8c1f1028123a.png)

**Comment by damienlong** [comment]
I’m an internal employee. The reason I raised it was that I was on the next version and don’t believe I’m on an insiders program. I was just asking whether CoE authors were on next version then customers.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Jenefer-Monroe** [comment]
Got it. I am not a MSFT employee and so I do not have access to those bits either, but other CoE devs are so that may be the case.
I am able to use the Sept version of the kit but I believe I just ignored this error message. In the future we will watch for these and ensure its not there when we shp.

**Comment by peteredc** [comment]
So... is there a location to get the Sept release of Power BI?  Or is this a glitch on the authors that used the Sept release for the PBI Report and only the Aug PBI Desktop is GA?

**Comment by Jenefer-Monroe** [comment]
There sure is. Here is the release page for the two months in quesion:
August: https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-August2022
September: https://github.com/microsoft/coe-starter-kit/releases/tag/CoEStarterKit-September2022

If you go to the bottom of each page you will see a place to download the kit for that month
![image](https://user-images.githubusercontent.com/89584709/189982776-d00613cc-0c65-461c-b143-ed922aa57293.png)

And in side you will find the last iteration we do for a month (if we do multiple) for each asset. Here shows the PBIT
![image](https://user-images.githubusercontent.com/89584709/189982910-bba46185-1590-4b77-b34d-6b0c22a984d4.png)

Please let me know if this makes sense or if I missed your question.

**Comment by peteredc** [comment]
Sorry what I meant is, is the Sept release of Power BI Desktop available somewhere considering the Sept CoE Release PBI Report is saved in Sept Power BI Desktop?

**Comment by Jenefer-Monroe** [comment]
Oh apologies. It's pretty clear thats what you were asking now that I re-read. 
I dont think there is no, but it worked for me when I just ignored the warning. Let me try again but have to tried to ignore the warning?

**Comment by Jenefer-Monroe** [comment]
So this works for me. Let me know if it works for you. 
1. Boot the Sept PBIT. Ignore this error (we'll be more careful in the future)
![image](https://user-images.githubusercontent.com/89584709/189984391-ddf0a408-685b-4fa3-bf0e-5f667fe08880.png)

2. Enter your URL and click Load
![image](https://user-images.githubusercontent.com/89584709/189984588-450b8398-c956-4588-ad38-cb7ef62bc4a9.png)

3. Wait
![image](https://user-images.githubusercontent.com/89584709/189984671-f1352361-a69d-4f72-b8da-70e7d61d1969.png)
![image](https://user-images.githubusercontent.com/89584709/189984726-c21672fe-872a-4204-873f-119759259000.png)
![image](https://user-images.githubusercontent.com/89584709/189984920-149af15c-3cab-4e55-80da-342f947f3f15.png)

4. It all loads up
![image](https://user-images.githubusercontent.com/89584709/189985001-ac04b88a-9768-42a8-9614-2393243e69c4.png)

**Comment by peteredc** [comment]
This does work.  However as you mentioned it's not ideal! :)

**Comment by Jenefer-Monroe** [comment]
Good yes. Apologies. We will make sure to save as an identity that isnt forward in PBI builds going forward.

**Comment by peteredc** [comment]
Well the MS download link is August.  So it would be good to know where you can download the Sept version.  Clearly it exists!
________________________________
From: Damien Long ***@***.***>
Sent: September 13, 2022 8:41 AM
To: microsoft/coe-starter-kit ***@***.***>
Cc: Byrne, Peter (External Consultant) ***@***.***>; Author ***@***.***>
Subject: [EXTERNE]Re: [microsoft/coe-starter-kit] [CoE Starter Kit - BUG] Version issue when opening Sept CoE Dashboard PBI report (Issue #3655)



EXTERNAL EMAIL – USE CAUTION

COURRIEL EXTERNE – FAITES PREUVE DE PRUDENCE


I’m an internal employee. The reason I raised it was that I was on the next version and don’t believe I’m on an insiders program. I was just asking whether CoE authors were on next version then customers.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
